src/PostEventHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { SQSEvent, SQSRecord } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { Constants } from "./utils/Constants";
+import { HttpCodesEnum } from "./models/enums/HttpCodesEnum";
+import { PostEventProcessor } from "./services/PostEventProcessor";
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : Constants.IPR_METRICS_NAMESPACE;
```

Comment:
We're attempting to pull from the environment here, but always seem to fall back to constants, and dont set the environment variables in the infrastructure. 
This will make debugging issues with this data harder because the source and provenance of these values are uncertain. These should be set in the [application configuration] if they can be set based on environment (https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3477274625/DRAFT+-+F2F+Application+Configuration) & checked to only return those values. Ideally we'd remove the logic around them entirely.

Reply:
We set it in the env vars [here](https://github.com/alphagov/di-ipvreturn-api/blob/main/deploy/template.yaml#L164) right? Agreed I think we shouldn't have it in both env vars and constants it'll make it hard to know what to look for instead we should have it in env vars and add check to ensure it's avail and throw error if not